recipe/meta.yaml

```diff
@@ -1,30 +1,39 @@
 
+{% set name = "usgscsm" %}
+{% set version = "main" %}
+
 package:
-  name: usgscsm
-  version: {{ environ.get("GIT_DESCRIBE_TAG", "") }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
```

Comment:
I imagine we would also need a PR to the feedstock to update the recipe?

Reply:
Yes, this was just for internal testing that is similar to the feedstock. That's the next thing I will have to tackle once I pull a new release